docs/section10/10-15.md

```diff
@@ -0,0 +1,59 @@
+# Airside systems – Rule 10-15  
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True    
+**Rule ID:** 10-15    
+ 
+**Rule Description:** The proposed design includes humidification and the baseline building design has been modeled with humidification. 
+
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE                                              
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each Zone   
+
+**Applicability Checks:** 
+
+1. The zone in the P-RMR includes humidification.
+
+**Function Call:** 
+
+1. match_data_element()
+2. get_baseline_system_types() 
+3. get_list_hvac_systems_associated_with_zone()
+4. get_hvac_zone_list_w_area()
+
+
+**Rule Logic:**  
+**Applicability Check 1:** 
+- Get dictionary list of proposed zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_p = get_hvac_zone_list_w_area (P_RMR)`
+- Get dictionary for finding baseline system types for hvac systems: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
```

Comment:
It seems like ```baseline_hvac_sys_type_ids_dict_b``` is not used anywhere below. Could you double-check this?

Reply:
I agree. I do not see this used anywhere.

Reply:
Thanks! Should we delete the line if not needed?

Reply:
Sounds good!